From a2f7648ff2cd6d7545c2b56ddc698e550b57d423 Mon Sep 17 00:00:00 2001
From: Khiem Nguyen <khiem.nguyen.xt@renesas.com>
Date: Wed, 10 Mar 2021 17:35:49 +0700
Subject: [PATCH 24/40] drm: rcar-rvgc: Add lacked header files

There are changes of header file since v5.0
as commit fcd70cd36 (drm: Split out drm_probe_helper.h)
and commit 7d0b0ada (drm/rcar-du: prepare for drmP.h removal from
drm_modeset_helper.h)
This patch added the lacked header files.

Signed-off-by: Khiem Nguyen <khiem.nguyen.xt@renesas.com>
---
 drivers/gpu/drm/rcar-rvgc/rcar_rvgc_kms.c  | 2 ++
 drivers/gpu/drm/rcar-rvgc/rcar_rvgc_pipe.c | 3 +++
 2 files changed, 5 insertions(+)

diff --git a/drivers/gpu/drm/rcar-rvgc/rcar_rvgc_kms.c b/drivers/gpu/drm/rcar-rvgc/rcar_rvgc_kms.c
index 3a8c80cc51f6..f909e49955a9 100644
--- a/drivers/gpu/drm/rcar-rvgc/rcar_rvgc_kms.c
+++ b/drivers/gpu/drm/rcar-rvgc/rcar_rvgc_kms.c
@@ -3,6 +3,8 @@
 #include <drm/drm_atomic_helper.h>
 #include <drm/drm_fb_cma_helper.h>
 #include <drm/drm_gem_framebuffer_helper.h>
+#include <drm/drm_probe_helper.h>
+#include <drm/drm_vblank.h>
 
 #include <linux/kthread.h>
 #include <linux/delay.h>
diff --git a/drivers/gpu/drm/rcar-rvgc/rcar_rvgc_pipe.c b/drivers/gpu/drm/rcar-rvgc/rcar_rvgc_pipe.c
index 17af33642f16..ca44fa3bf1fc 100644
--- a/drivers/gpu/drm/rcar-rvgc/rcar_rvgc_pipe.c
+++ b/drivers/gpu/drm/rcar-rvgc/rcar_rvgc_pipe.c
@@ -11,6 +11,9 @@
 /* TODO: Need plane helper ? */
 #include <drm/drm_fb_cma_helper.h>
 #include <drm/drm_gem_cma_helper.h>
+#include <drm/drm_vblank.h>
+#include <drm/drm_drv.h>
+#include <drm/drm_probe_helper.h>
 
 #include "rcar_rvgc_taurus.h"
 #include "r_taurus_rvgc_protocol.h"
-- 
2.25.1

